pkg/types/aws/platform.go

```diff
@@ -144,6 +149,92 @@ type ServiceEndpoint struct {
 	URL string `json:"url"`
 }
 
+// VPCSpec configures the VPC for the cluster.
+type VPCSpec struct {
```

Comment:
I think we should just call this `VPC` or at least not have the `Spec` suffix on it, because it's not really a spec...

Reply:
Right, make sense to me!